src/frontend/web_application/src/components/MessageList/components/Message/index.jsx

```diff
@@ -1,188 +1,18 @@
-        isExpanded: false,
-        body: body.substring(0, 140),
-      });
-    } else {
-      this.setState({
-        body,
-      });
-    }
-  }
-
-  handleActiveClick() {
-    this.setState(prevState => ({
-      isActive: !prevState.isActive,
-    }));
-  }
-
-  handleExpandClick() {
-    const message = this.props.message;
-    const body = message.body;
-    this.setState(prevState => ({
-      isExpanded: !prevState.isExpanded,
-      body: prevState.isExpanded ? body.substring(0, 140) : body,
-    }));
-  }
-
-  render() {
-    const { message, __ } = this.props;
-    const author = message.participants.find(participant => participant.type === 'From');
-    const subject = message.subject;
-    const topBarClassName = classnames(
-      'm-message__top-bar',
-      { 'm-message__top-bar--active': this.state.isActive }
-    );
-    const bodyClassName = classnames(
-      'm-message__body__content',
-      { 'm-message__body__content--expanded': this.state.isExpanded }
-    );
-
-    const dropdownId = uuidV1();
-
-    return (
-      <div className="m-message">
-        <div className="m-message__avatar-col">
-          <ContactAvatarLetter
-            contact={author}
-            className="m-message__avatar"
-          />
-        </div>
-        <div className="m-message__content">
-
-          <div className={topBarClassName}>
-            <MessageInfosContainer
-              message={message}
-              author={author}
-              __={__}
-            />
-
-            <DropdownControl toggle={dropdownId} className="m-message__actions-switcher">
-              <Icon type="ellipsis-v" />
-            </DropdownControl>
-
-            <Dropdown
-              id={dropdownId}
-              className="m-message__actions-menu"
-              position="left"
-              closeOnClick
-            >
-              <MessageActionsContainer
-                className="m-message__actions-container"
-                __={__}
-              />
-            </Dropdown>
-
-          </div>
-
-          <div className="m-message__body">
-            {subject &&
-              <div className="m-message__body__subject">{subject}</div>
-            }
-            <div className={bodyClassName}>
-              {this.state.body}
-            </div>
-          </div>
-
-          {this.state.isTooLong &&
-            <div className="m-message__expand-button">
-              <Button
-                onClick={this.handleExpandClick}
-                value={this.state.isExpanded}
-                title={this.state.isExpanded ? __('message-list.message.action.collapse') : __('message-list.message.action.expand')}
-              >
-                {this.state.isExpanded ? __('message-list.message.action.collapse') : __('message-list.message.action.expand')}
-              </Button>
-            </div>
-          }
-
-        </div>
-      </div>
-    );
-  }
-}
-
-export default Message;
+import { createSelector } from 'reselect';
```

Comment:
ok be carefull, I suppose it is the source of the conflict.

The final file looks good here